decidim-core/lib/decidim/core/test.rb

```diff
@@ -21,3 +21,4 @@
 require "decidim/core/test/shared_examples/follows_examples"
 require "decidim/core/test/shared_examples/simple_event"
 require "decidim/core/test/shared_examples/component_type"
+require "decidim/comments/test"
```

Comment:
We can't add this here since this creates a dependency from core to comments

Reply:
Ok, which is the correct way to make all the examples in decidim/comments/test/shared_examples available to proposals?
I've moved the `require` into `decidim-proposals/spec/spec_helper`.